ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -192,7 +188,7 @@ metadata:
   namespace: CLUSTERNAMESPACE
 spec:
   clusterName: CLUSTERNAME
-  replicas: 2
+  replicas: 3
```

Comment:
This should be 2. This is because test-vmc.sh expects exactly 2 worker nodes:

```
234         for t in {1..10}
235         do
236             sleep 1m
237             echo "=== Get node (try for 1m) ==="
238             mdNum="$(kubectl get node | grep -c ${CLUSTER}-md)"
239             if [ "${mdNum}" == "2" ]; then
240                 echo "=== Setup workload cluster succeeded ==="
241                 CLUSTER_READY=true
242                 break
243             fi
244         done
```

Reply:
Updated